src/Aspire.Hosting/Dcp/Process/ProcessUtil.cs

```diff
@@ -108,6 +93,21 @@ public static (Task<ProcessResult>, IAsyncDisposable) Run(ProcessSpec processSpe
             process.BeginOutputReadLine();
             process.BeginErrorReadLine();
             processSpec.OnStart?.Invoke(process.Id);
+
+            process.WaitForExitAsync().ContinueWith(t =>
+            {
+                startupComplete.Wait();
+
+                if (processSpec.ThrowOnNonZeroReturnCode && process.ExitCode != 0)
+                {
+                    processLifetimeTcs.TrySetException(new InvalidOperationException(
+                        $"Command {processSpec.ExecutablePath} {processSpec.Arguments} returned non-zero exit code {process.ExitCode}"));
+                }
+                else
+                {
+                    processLifetimeTcs.TrySetResult(new ProcessResult(process.ExitCode));
+                }
+            }, TaskScheduler.Current).ConfigureAwait(false);
```

Comment:
```suggestion
            }, TaskScheduler.Current);
```

You don't need to call ConfigureAwait because you aren't awaiting this task.

Reply:
I believe I was getting a warning/error based on what I assumed to be a repo-mandated linting rule, but I'll double check.

Reply:
I probably had an await at some point and saw the code analysis error. Removed.